Tchou-Tchou/StatusMenuController.swift

```diff
@@ -4,49 +4,67 @@ import CoreWLAN
 class StatusMenuController: NSObject {
     @IBOutlet var statusMenu: NSMenu!
     let statusItem = NSStatusBar.system.statusItem(withLength: NSStatusItem.variableLength)
-
+    
     let wifiAPI = WifiAPI()
-
+    
     @IBAction func clickedQuit(_ sender: NSMenuItem) {
         NSApplication.shared.terminate(self)
     }
-
+    
     override func awakeFromNib() {
         statusItem.menu = statusMenu
         statusItem.highlightMode = true
         statusItem.configure(with: nil)
-
+        
         refreshSpeed()
     }
-
+    
     @objc func refreshSpeed() {
+        let supportedNetworks = [
+            [
+                "ssid" : "_SNCF_WIFI_INOUI",
+                "unit" : UnitSpeed.metersPerSecond
+            ],
+            [
+                "ssid" : "WIFIonICE",
+                "unit" : UnitSpeed.kilometersPerHour
+            ]
+        ]
+        let supportedSsids = supportedNetworks.flatMap { supportedNetwork in supportedNetwork["ssid"] } as! [String]
+        let currentSsid = CWWiFiClient.shared().interface()?.ssid()
```

Comment:
if you guard against current SSID nullity, you won't have to force unwrap it in the future (for example in the call of `fetchSpeed()`)
```
guard let currentSsid = CWWiFiClient.shared().interface()?.ssid() else { 
    self.launchTimer()
    return 
}
```